README.md

```diff
@@ -1,12 +1,16 @@
 <p style="text-align:center;" align="center">
-  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/assets/images/layer5/layer5-tag-community-white-bg.png" width="45%" /></p>
-  
+  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/src/assets/images/layer5/layer5-tagline/png/layer5-tag-community-white-bg.png" width="45%" /></p>
```

Comment:
@leecalcote : Here using, Layer5 logo only. Take this to your end :slightly_smiling_face: